### ISSUE TITLE
resource hsdp_dicom_remote_node does not take port, hostname, ip_address in network_connection as input variables

### ISSUE THREAD

**Issue by kishoreinvits** [philips-software/terraform-provider-hsdp#109]
When using resource `hsdp_dicom_remote_node`,  port, hostname, ip_address in network_connection are not taken properly.
When debug logs are enabled, I see below error. From request body, I see it is not taking input I am passing at all.

```
[go-hsdp-api 8387ccf1-ed66-41a0-969d-c2583cc62013-1] --- Request start ---
POST /store/dicom/config/dicom/production/remoteNodes HTTP/1.1
Host: dss-config-edisa-tst.us-east.philips-healthsuite.com
Accept: application/json
Api-Version: 1
Authorization: [sensitive]
Content-Type: application/json
User-Agent: go-hsdp-api/dicom/0.42.1

{"title":"orthanc","networkConnection":{"port":0,"hostName":"","ipAddress":"","disableIpv6":false,"advancedSettings":{"pduLength":0,"artimTimeOut":0,"associationIdleTimeOut":0,"certificateInfo":{"id":""},"authenticateClientCertificate":false},"networkTimeout":0,"isSecure":false},"aeTitle":"ORTHANC"}
[go-hsdp-api 8387ccf1-ed66-41a0-969d-c2583cc62013-1] Request end ---
[go-hsdp-api 8387ccf1-ed66-41a0-969d-c2583cc62013-1] --- Response start ---
HTTP/1.1 422 Unprocessable Entity
Content-Length: 469
Connection: keep-alive
Content-Type: application/json
Date: Mon, 30 Aug 2021 10:13:17 GMT
X-Vcap-Request-Id: ce20faec-ae67-4dff-53a8-e1444090d2bd

{"error":"Request body is not parsable. :NoAdditionalPropertiesAllowed: #/networkConnection.advancedSettings.certificateInfo\nNoAdditionalPropertiesAllowed: #/networkConnection.advancedSettings.authenticateClientCertificate\nPatternMismatch: #/networkConnection.hostName\nStringTooShort: #/networkConnection.hostName\nNotOneOf: #/networkConnection.ipAddress\n{\n  IpV4Expected: #/networkConnection.ipAddress\n}\n{\n  IpV6Expected: #/networkConnection.ipAddress\n}\n\n"}
[go-hsdp-api 8387ccf1-ed66-41a0-969d-c2583cc62013-1] --- Response end ---

```
Observed this on version [0.18.9](https://registry.terraform.io/providers/philips-software/hsdp/0.18.9)

**Comment by loafoe** [comment]
This should be fix in an upcoming release, the read out of the `network_connection` fields was missing

**Comment by loafoe** [comment]
Fixed in `v0.19.0`

**Comment by kishoreinvits** [comment]
> Fixed in `v0.19.0`

Will use this version. Thanks @loafoe!
